sections/queueing_model/proportion_within_target/main.tex

```diff
@@ -0,0 +1,141 @@
+Similar to section \ref{sec:waiting_time} three formulas are needed for this 
+performance measure.
+
+The proportion of type 1 individuals within a time target:
+
+\begin{equation}\label{eq:proportion_within_target_type_1}
+    P(X^{(1)} < t) = \frac{\sum_{(u,v) \in S_A^{(1)}} P(X_{(u,v)}^{(1)} < t) 
+    \pi_{u,v} }{\sum_{(u,v) \in S_A^{(1)}} \pi_{u,v}}
+\end{equation}
+
+The proportion of type 2 individuals within a time target:
+
+\begin{equation}\label{eq:proportion_within_target_type_2}
+    P(X^{(2)} < t) = \frac{\sum_{(u,v) \in S_A^{(2)}} P(X_{(u,v)}^{(2)} < t) 
+    \pi_{u,v} }{\sum_{(u,v) \in S_A^{(2)}} \pi_{u,v}}
+\end{equation}
+
+The overall proportion individuals within a time target (where \(P_{L'_1}\) and 
+\(P_{L'_1}\) are defined in (\ref{eq:proportion_of_accepting_individuals})):
+
+\begin{align}
+    P(X < t) &= \frac{\lambda_1 P_{L'_1}}{\lambda_2 P_{L'_2}+\lambda_1 P_{L'_1}} 
+    P(X^{(1)} < t) \nonumber \\
+    &+ \frac{\lambda_2 P_{L'_2}}{\lambda_2 P_{L'_2} + \lambda_1 
+    P_{L'_1}} P(X^{(2)} < t) \label{eq:overall_proportion_within_target}
+\end{align}
+
+Here \(P(X_{(u,v)}^{(1)})\) and \(P(X_{(u,v)}^{(2)})\) are defined as the
+proportion of individuals within the time target \(t\) when starting from state 
+\((u,v)\).
+These expression can be calculated by:
+
+\begin{equation}\label{eq:proportion_within_target_type_1_from_state}
+    P(X_{(u,v)}^{(1)} < t) = 
+    \begin{cases}
+        1 - \sum_{i=0}^{v-1} \frac{1}{i!} e^{-\mu t} (\mu t)^i, 
+            & \textbf{if } C = 1 \\
+            & \textbf{and } v>1 \\
+        & \\
+        1 - (\mu C)^{v-C} \mu  
+            \sum_{k=1}^{\mid \vec{r} \mid} \sum_{l=1}^{r_k}
+            \frac{\Psi_{k,l}(-\lambda_k)t^{r_k - l} 
+            e^{-\lambda_k t}}{(r_k - l)! (l - 1)!},
+            & \textbf{if } C > 1 \\
+        \textbf{where } \vec{r}=(v - C, 1) \textbf{ and } 
+            \vec{\lambda}=(C \mu, \mu) & \textbf{and } v > C \\
+            \hspace{1.15cm} \lambda_0 = 0, r_0 = 1 \\
+        & \\
+        1 - e^{-\mu t},  & \textbf{if } v \leq C
+    \end{cases}
+\end{equation}
+
+
+\begin{equation}\label{eq:proportion_within_target_type_2_from_state}
+    P(X_{(u,v)}^{(2)} < t) = 
+    \begin{cases}
+        1 - \sum_{i=0}^{\min(v,T)-1} \frac{1}{i!} e^{-\mu t} (\mu t)^i,  
+            & \textbf{if } C = 1 \\ 
+            & \textbf{and } v, T > 1 \\
+            & \\
+        1 - (\mu C) ^ {\min(v,T) - C} \mu  
+            &  \\
+        \qquad \times \sum_{k=1}^{\mid \vec{r} \mid} \sum_{l=1}^{r_k} 
+        \frac{\Psi_{k,l}(-\lambda_k)t^{r_k - l} 
+        e^{-\lambda_k t}}{(r_k - l)! (l - 1)!}, 
+            & \textbf{if } C > 1 \\
+        \textbf{where } \vec{r}=(\min(v, T) - C, 1) 
+            & \textbf{and } v, T  > C\\
+        \hspace{1.15cm} \vec{\lambda}=(C \mu, \mu) \\
+        \hspace{1.15cm} \lambda_0 = 0, r_0 = 1 \\
+            & \\
+        1 - e^{-\mu t}, & \textbf{if } v \leq C \\ 
+            & \textbf{or } T \leq C \\
+    \end{cases}
```

Comment:
I started making LaTeX suggestions but it got messy:

- Remove the extra blank lines you add.
- Move the `\textbf{where} ...` statement to outside of the equation.

Reply:
Will do.